packages/text/src/vars/index.ts

```diff
@@ -1,5 +1,19 @@
 import { keyMirror } from '@pluralsight/ps-design-system-util'
 
-export const headingSizes = keyMirror('smallCaps', 'medium', 'large', 'xLarge')
+export const capsSizes = keyMirror('small', 'medium', 'large')
```

Comment:
Pretty personal opinion, but the "large" cap size isn't large at all.

For the flexibility of the system in the future, I wonder if we shouldn't choose names that are visually smaller, so that if our design changes in the future where we actually want large caps, we'll have a scale that is easier to adjust in that direction.

Maybe `xXSmall`, `xSmall` and `small`?

Reply:
The size names are relative to the category of the type style rather than being relative to all of the type styles. "large" is large for all caps text.

Reply:
👍 Totally makes sense.  Was more a forward-looking potential adjustment for if the category was ever expanded.